src/TypeDefinition.js

```diff
@@ -311,9 +311,10 @@ export type NavigationDrawerScreenOptions = NavigationScreenOptions & {
     >),
   drawerLabel?:
     | React.Element<*>
-    | ((options: { tintColor: ?string, focused: boolean }) => ?React.Element<
-      *
-    >),
+    | ((
+      options: { tintColor: ?string, focused: boolean },
+    ) => ?React.Element<*>),
+  drawerLockMode?: 'unlocked' | 'locked-close' | 'locked-open', // defaults `unlocked`
```

Comment:
It would be 'locked-closed' instead of 'locke-close': https://facebook.github.io/react-native/docs/drawerlayoutandroid.html#drawerlockmode